lib/features/home/widgets/googe_map.dart

```diff
@@ -55,27 +73,30 @@ class _GoogleMapViewState extends State<GoogleMapView>
             : BitmapDescriptor.fromBytes(data);
 
         return AnimatedBuilder(
-            animation: _controller,
-            builder: (BuildContext context, Widget? child) => GoogleMap(
-                    initialCameraPosition: CameraPosition(
-                        target: LatLng(widget.currentLocation.latitude,
-                            widget.currentLocation.longitude),
-                        zoom: 15),
-                    markers: <Marker>{
-                      Marker(
-                          markerId: const MarkerId('currentLocation'),
-                          position: widget.currentLocation,
-                          icon: icon,
-                          onTap: () {
-                            if (_controller.value == 1) {
-                              _controller.reverse();
-                            } else {
-                              _controller.forward();
-                            }
-                          })
-                    }));
+          animation: _controller,
+          builder: (BuildContext context, Widget? child) => GoogleMap(
+              onMapCreated: (GoogleMapController controller) =>
+                  mapController = controller,
+              initialCameraPosition: CameraPosition(
+                  target: LatLng(widget.currentLocation.latitude,
```

Comment:
`widget.currentLocation` is a LatLng object, you don't need to do this conversion